shared/src/sources/balancer/swap/fixed_point.rs

```diff
@@ -34,7 +34,6 @@ lazy_static! {
     static ref MAX_POW_RELATIVE_ERROR: Bfp = Bfp(10000_usize.into());
 }
 
-#[cfg(test)]
```

Comment:
I don't think this is needed right?

Reply:
It was the only way I could use `Bfp::from(usize)` in a test elsewhere.

Reply:
Ah, because its needed outside of the `shared` crate. Makes sense.